src/ExposureHistoryContext.tsx

```diff
@@ -87,6 +87,18 @@ const ExposureHistoryProvider: FunctionComponent<ExposureHistoryProps> = ({
     })
   }, [])
 
+  const getExposureStatus = useCallback(() => {
+    GetStoreData(NEW_EXPOSURES).then((exposureStatus) => {
```

Comment:
is there a way we can do this without using async storage?

Reply:
@johnschoeman This is based on a conversation in the old repo: https://github.com/Path-Check/covid-safe-paths/pull/1286#issuecomment-660332673

The desired behavior is that the new exposure badge should be persisted if a user force quits the app without clearing the badge notification. Is there somewhere else you would prefer to persist this state?

Reply:
nope, makes sense to me was just wondering.

Reply:
although I was chatting with @tparesi about this, are we clear on what the 'last updated at' should be? is it the last time we received an actual exposure? or is it the last time we checked for an exposure?

I think if its the last time we checked we need the async storage as thats where the data is saved for the batch file processing.

If its the last time we received a possible exposure notification, could we update the lastDetectionDate when we consume the emitted exposure event from the native layer?  That would allow us to not need do an extra ping to the native side.

Reply:
@johnschoeman This might be worth some IRL context sharing as you mentioned, but if I understand you correctly, it seems like we are slightly conflating the concepts of "last updated at" and "new exposure badge". IIRC, "last updated at" represents the last time that PathCheck communicated with the GAEN server to see if there was a new exposure. Whereas the "new exposure badge" represents that the PathCheck check for exposures process completed, **and** that a new exposure was found, **and** that that happened since the last time that a user visited the ExposureHistory screen.

This leads me to believe that these two concepts are separate, and that it needs to have access to the JavaScript layer, because it is dependent on the user interacting with the ExposureHistory tab.